File: 217_contains_duplicate/217_contains_duplicate.ts
function containsDuplicate(nums: number[]): boolean {
  let hashDuplicate: Set<number> = new Set();

  for (let num of nums) {
    if (hashDuplicate.has(num)) {
      return true;
    }
    hashDuplicate.add(num);
  }
  return false;
}

File: 217_contains_duplicate/217_contains_duplicate.go
package containsduplicate

func containsDuplicate(nums []int) bool {
	hashDup := make(map[int]int)
	for _, num := range nums {
		if hashDup[num] == 1 {
			return true
		}
		hashDup[num] = 1
	}
	return false
}
